experimental Nix feature 'nix-command' is disabled

Because the action is calling `nix path-info`, it might fail in some Nix installations.

```
/bin/sh -c nix path-info --all | grep -v '.drv$' > /tmp/store-path-pre-build
error: --- Error --- nix
experimental Nix feature 'nix-command' is disabled; use '--experimental-features nix-command' to override
##[error]Action failed with error: Error: The process '/bin/sh' failed with exit code 1
(node:3014) UnhandledPromiseRejectionWarning: Error: The process '/bin/sh' failed with exit code 1
    at ExecState._setResult (/home/runner/work/_actions/cachix/cachix-action/v6/dist/main/index.js:907:25)
    at ExecState.CheckComplete (/home/runner/work/_actions/cachix/cachix-action/v6/dist/main/index.js:890:18)
    at ChildProcess.<anonymous> (/home/runner/work/_actions/cachix/cachix-action/v6/dist/main/index.js:790:27)
    at ChildProcess.emit (events.js:210:5)
    at maybeClose (internal/child_process.js:1021:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)
(node:3014) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:3014) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

This is only an issue when using https://github.com/numtide/nix-flakes-installer

--- a/src/main.ts
+++ b/src/main.ts
@@ -45,7 +45,7 @@ async function setup() {
       core.exportVariable('CACHIX_SIGNING_KEY', signingKey);
     }
     // Remember existing store paths
-    await exec.exec("sh", ["-c", `nix path-info --all | grep -v '\.drv$' > /tmp/store-path-pre-build`]);
+    await exec.exec("sh", ["-c", `${__dirname}/list-nix-store.sh > /tmp/store-path-pre-build`]);
   } catch (error) {
     core.setFailed(`Action failed with error: ${error}`);
     throw (error);

